wordpress/containerbuddy/onstart.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# Refresh all config files in order
+#
+# This script is typically called once at the container start, but
+# it can be called manually if WP config details must be changed
+
+# The database and memcached config files are separate to avoid collisions
+# if their backends' onChange handlers are triggered simultaneously
+echo "******running onstart script*********"
+
+until [[ `curl -s consul:8500/v1/health/state/passing | grep mysql-primary` ]]
```

Comment:
oh nice, thanks for digging that up. I wasn't really happy with the way I had to implement this, but I was struggling with parsing the json in pure bash (one of the reasons I originally was working in python). I toyed with the idea of adding https://stedolan.github.io/jq/ to help with json in bash, but I wasn't sure if I wanted to add another dependency.